website/server/controllers/top-level/dataexport.js

```diff
@@ -97,7 +112,26 @@ async function _getUserDataForExport (user) {
  * @apiGroup DataExport
  * @apiDescription NOTE: Part of the private API that may change at any time.
  *
- * @apiSuccess {String} A json file
```

Comment:
@crookedneighbor That might be a good idea, but it also adds tasks organized by type, which is not part of the get user route